homeassistant/components/verisure/climate.py

```diff
@@ -0,0 +1,218 @@
+            self.heatpumpstate, '$.latestClimateSample.temperature')[0]
+        if self._config_date is None or self._config_date < sync_date:
+            self._target_temperature = self.jsonpath(
+                self.heatpumpstate, '$.heatPumpConfig.targetTemperature')[0]
+            current_operation = self.jsonpath(
+                self.heatpumpstate, '$.heatPumpConfig.mode')[0]
+            self._current_operation = VERISIRE_HASS_OP_MODE[current_operation]
+            self._current_fan_mode = self.jsonpath(
+                self.heatpumpstate, '$.heatPumpConfig.fanSpeed')[0].title()
+            self._current_swing_mode = self.jsonpath(
+                self.heatpumpstate,
+                '$.heatPumpConfig.airSwingDirection.vertical')[0].title()
+            self._on = bool(self.jsonpath(
+                self.heatpumpstate,
+                '$.heatPumpConfig.power')[0] == 'ON')
+            self._config_date = sync_date
+
+    @property
+    def supported_features(self):
+        """Return the list of supported features."""
+        return self._support_flags
+
+    @property
+    def should_poll(self):
+        """Return the polling state."""
+        return True
+
+    @property
+    def name(self):
+        """Return the name of the climate device."""
+        return self._name
+
+    @property
+    def target_temperature_step(self):
+        """Representation target temperature step."""
+        return 1.0
+
+    @property
+    def temperature_unit(self):
+        """Return the unit of measurement."""
+        return self._unit_of_measurement
+
+    @property
+    def current_temperature(self):
+        """Return the current temperature."""
+        return self._current_temperature
+
+    @property
+    def target_temperature(self):
+        """Return the temperature we try to reach."""
+        return self._target_temperature
+
+    @property
+    def current_operation(self):
+        """Return current operation ie. heat, cool, idle."""
+        return self._current_operation
+
+    @property
+    def operation_list(self):
+        """Return the list of available operation modes."""
+        return self._operation_list
+
+    @property
+    def is_on(self):
+        """Return true if the device is on."""
+        return self._on
+
+    @property
+    def current_fan_mode(self):
+        """Return the fan setting."""
+        return self._current_fan_mode
+
+    @property
+    def fan_list(self):
+        """Return the list of available fan modes."""
+        return self._fan_list
+
+    def set_temperature(self, **kwargs):
+        """Set new target temperatures."""
+        if self._on:
+            if kwargs.get(ATTR_TEMPERATURE) is not None:
+                self._target_temperature = kwargs.get(ATTR_TEMPERATURE)
+                hub.session.set_heat_pump_target_temperature(
+                    self.heatpump_id, self._target_temperature)
+        self.schedule_update_ha_state()
```

Comment:
We don't need to schedule a state update here since this is a polling entity. A state update will be scheduled at the end of the service call.